docs/devGuide/development/settingUp.md

```diff
@@ -32,10 +32,6 @@ This page explains how to set up your development environment to start contribut
    * `dot -V` (for Graphviz - optional on Windows)
    * `python3 -V`
 
-<box type="warning" seamless>
-
-With Node LTS (v20), the python used has updated to 3.12, and [you may get an error](https://github.com/MarkBind/markbind/issues/2496). You can solve this issue by installing setuptools or deprecating the versions used.
```

Comment:
Is this no longer relevant?

Reply:
yeap! #2496 was resolved by #2565 as the bug was caused by the outdated dev dependency

Reply:
I see, thanks! We could have updated the docs within the PR https://github.com/MarkBind/markbind/pull/2565.